server/src/main/java/org/elasticsearch/index/mapper/KeywordFieldMapper.java

```diff
@@ -232,29 +230,16 @@ public Builder(final String name, final MappingParserContext mappingParserContex
             String name,
             IndexAnalyzers indexAnalyzers,
             ScriptCompiler scriptCompiler,
-            int ignoreAboveDefault,
             IndexVersion indexCreatedVersion,
             SourceKeepMode sourceKeepMode
         ) {
-            this(
-                name,
-                indexAnalyzers,
-                scriptCompiler,
-                ignoreAboveDefault,
```

Comment:
`ignoreAboveDefault` was removed, and this entire code block was condensed into a single line